pkg/env/sensor.go

```diff
@@ -20,6 +20,12 @@ var (
 	// This is typically used for Sensor to communicate with a local Scanner-slim's gRPC server.
 	ScannerSlimGRPCEndpoint = RegisterSetting("ROX_SCANNER_GRPC_ENDPOINT", WithDefault("scanner.stackrox.svc:8443"))
 
+	// ScannerV4GRPCEndpoint is used to communicate with the Scanner V4 endpoint in the same cluster.
+	ScannerV4GRPCEndpoint = RegisterSetting("ROX_SCANNER_V4_GRPC_ENDPOINT", WithDefault("scanner-v4.stackrox.svc:8443"))
```

Comment:
Note: we will need to make sure this is aligned with the k8s configurations once those are done. Nothing needed to be done now, but something to consider later